app/config/locale/translations/ur.json

```diff
@@ -3,30 +3,30 @@
     "settings.locale": "ur",
     "settings.direction": "rtl",
     "emails.sender": "%s ٹیم",
-    "emails.verification.subject": "",
-    "emails.verification.hello": "",
-    "emails.verification.body": "",
-    "emails.verification.footer": "",
-    "emails.verification.thanks": "",
-    "emails.verification.signature": "",
-    "emails.magicSession.subject": "",
-    "emails.magicSession.hello": "",
-    "emails.magicSession.body": "",
-    "emails.magicSession.footer": "",
-    "emails.magicSession.thanks": "",
-    "emails.magicSession.signature": "",
-    "emails.recovery.subject": "",
-    "emails.recovery.hello": "",
-    "emails.recovery.body": "",
-    "emails.recovery.footer": "",
-    "emails.recovery.thanks": "",
-    "emails.recovery.signature": "",
-    "emails.invitation.subject": "",
-    "emails.invitation.hello": "",
-    "emails.invitation.body": "",
-    "emails.invitation.footer": "",
-    "emails.invitation.thanks": "",
-    "emails.invitation.signature": "",
+    "emails.verification.subject": "اکاؤنٹ کی تصدیق",
+    "emails.verification.hello": "خوش آمدید {{name}}",
+    "emails.verification.body": "براہ کرم اپنے ای میل کی تصدیق کے لیے درج ذیل لنک پر عمل کریں۔",
+    "emails.verification.footer": "اگر آپ نے اس پتے کی تصدیق کے لیے نہیں کہا تو آپ اس پیغام کو نظر انداز کر سکتے ہیں۔",
+    "emails.verification.thanks": "شکریہ",
+    "emails.verification.signature": "ٹیم۔ {{project}}",
+    "emails.magicSession.subject": "اگ ان کریں",
+    "emails.magicSession.hello": "خوش آمدید,",
+    "emails.magicSession.body": "لاگ ان کرنے کے لیے اس لنک پر عمل کریں۔",
+    "emails.magicSession.footer": "اگر آپ نے اس ای میل کا استعمال کرتے ہوئے لاگ ان کرنے کے لیے نہیں کہا تو آپ اس پیغام کو نظر انداز کر سکتے ہیں۔",
+    "emails.magicSession.thanks": "شکریہ",
+    "emails.magicSession.signature": "ٹیم۔ {{project}}",
+    "emails.recovery.subject": "پاس ورڈ ری سیٹ۔",
+    "emails.recovery.hello": "ہیلو {{name}}",
+    "emails.recovery.body": "{{project}} کا پاس ورڈ تبدیل کرنے کے لیے درج ذیل لنک پر عمل کریں",
+    "emails.recovery.footer": "اگر آپ نے اپنا پاس ورڈ دوبارہ ترتیب دینے کے لیے نہیں کہا تو آپ اس پیغام کو نظر انداز کر سکتے ہیں۔",
+    "emails.recovery.thanks": "شکریہ",
+    "emails.recovery.signature": "ٹیم۔ {{project}}",
+    "emails.invitation.subject": "٪ s پر٪ s ٹیم کو دعوت",
```

Comment:
You need to use the right `%s` characters here

Reply:
thank you for pointing this out